packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3878,8 +3878,8 @@ class SemanticsConfiguration {
   /// This is a request to increase the value represented by the widget. For
   /// example, this action might be recognized by a slider control.
   ///
-  /// If [this.value] is set, [increasedValue] must also be provided and
-  /// [onIncrease] must ensure that [this.value] will be set to
+  /// If `this.value` is set, [increasedValue] must also be provided and
+  /// [onIncrease] must ensure that `this.value` will be set to
```

Comment:
I think this should be:

```suggestion
  /// If [value] is set, [increasedValue] must also be provided and
  /// [onIncrease] must ensure that [value] will be set to
```

Reply:
Done